python/lsst/obs/ctio0m9/ctio0m9Mapper.py

```diff
@@ -258,6 +258,12 @@ def std_dark(self, item, dataId):
             exp.getInfo().setVisitInfo(afwImage.VisitInfo(darkTime=1.0))
         return exp
 
+    def _setFilter(self, mapping, item, dataId):
+        """To ensure that ctio0m9 exposures get both physical and band set.
+        """
+        idFilter = mapping.need(['filter'], dataId)['filter']
+        item.setFilterLabel(afwImage.FilterLabel(physical=idFilter, band=idFilter))
+
```

Comment:
I don't understand this change at all. How is it better than `CameraMapper._setFilter`?

Reply:
obs_ctio0m9 doesn't have FilterDefinitions, so the "smart" lookup in the parent class doesn't work, and various downstream coded needed both physical and band to be set.

At the time I did this, ci_ctio0m9 was still being run by lsst_ci, but it has since been removed. I might as well merge this anyway.

Reply:
I see, and the fallback code does nothing if there's a partial label set.

However, you probably shouldn't be assuming in downstream code that both band and physical always exist -- what you're doing with a patch like this is fooling the code into acting on information it doesn't really have. IIRC the Gen 2 data IDs can be very flexible about which kind of filter they use, so a band listed as a physical filter and a physical filter listed as a band are _both_ possible outcomes here.

Reply:
Each bit of downstream code assumes a particular thing exists. I think in this case the failure was in the reference loader, which needs a band (which was None here because there's no FilterDefinitions). This particular package is no longer in lsst_distrib or lsst_ci, so it's all moot.

Reply:
I'm pretty sure using physical filter as the "band" will still cause problems for the reference loader.

If we're not going to support it, I suggest leaving the change unmerged. That way if anybody does try to use `obs_ctio0m9` with a recent stack, they'll at least get an obvious failure.

Reply:
This package uses the physical filter names (like `RONCHI200+z`) in the reference loader `filterMap`, so making them be the same thing is, I think, correct here. Because the refLoader expects `band`, I have to put something in `band`.

This change does make ci_ctio0m9 pass, so it's as correct as can be. I think I'd like to merge it just for that.

Reply:
Fine, but please include a comment explaining the compatibility implications.